gh-pages/_docs/06-04-custom-view.md

```diff
@@ -0,0 +1,10 @@
+---
+permalink: /docs/custom-views/
+title: "Custom View"
+---
+
+Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a "shortcut-button". In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
```

Comment:
The user could ask "Where can I find the shortcut-button?".

Reply:
I would suggest adapting the [user control documentation](https://maibornwolff.github.io/codecharta/docs/user-controls/) soon and then referring to it. That would have taken too much time now. I would be happy to open an issue for this @ce-bo ?

Reply:
Perfect, thank you!